templates/kubernetes/docs/supported-versions.md

```diff
@@ -177,7 +178,7 @@ channels:
   1.5/candidate:    1.5.5          2017-05-17    (3) 17MB -
   1.5/beta:         1.5.5          2017-05-17    (3) 17MB -
   1.5/edge:         1.5.5          2017-05-17    (3) 17MB -
-```
+
```

Comment:
` ``` ` This is still needed here right?

Reply:
I think this is actually a correction. What's below is clearly not code, but HTML content.
There are Vanilla classes that should be displayed
```
<!-- FEEDBACK -->
<div class="p-notification--information">
  <p class="p-notification__content">
    We appreciate your feedback on the documentation. You can
    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/main/pages/k8s/supported-versions.md" >edit this page</a>
    or
    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/issues/new" >file a bug here</a>.</p>
  </div>
</div>
```